cards.js

```diff
@@ -16,8 +17,12 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
 
-// Put some code in here!
 
-}
+function buildDeck(n){
+  for(var i = 0; i < n; i ++)
+    for(var j = 0; j < suits.length; j ++)
+      for(var k = 0; k < values.length; k ++)
+          new Card(values[k], suits[j]);
+  return buildDeck();
+  }
```

Comment:
for every `for` loop you execute you need to have `{}` wrapped around the code block for the for loop